### ISSUE TITLE
Accessibility: no focus state on notification links

### ISSUE THREAD

**Issue by joedolson** [WordPress/wp-feature-notifications#108]
### Description

Links within notification message should have a visible focus state. 

### Step-by-step reproduction instructions

Open notifications panel, use `tab` key to navigate links within. Observe lack of visible focus state.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except WP Feature Notifications.

Yes
